src/test/java/com/exasol/adapter/jdbc/JdbcAdapterTest.java

```diff
@@ -104,11 +85,27 @@ void testGetCapabilities() throws AdapterException {
                 createSchemaMetadataInfo());
         final ExaMetadata exaMetadataMock = mock(ExaMetadata.class);
         final GetCapabilitiesResponse response = this.adapter.getCapabilities(exaMetadataMock, request);
-        assertThat(response.getCapabilities().getMainCapabilities(), emptyCollectionOf(MainCapability.class));
+        assertThat(response.getCapabilities().getMainCapabilities(), contains(MainCapability.ORDER_BY_EXPRESSION));
     }
 
     @Test
-    void testDropVirtualSchemaMustSucceedEvenIfDebugAddressIsInvalid() throws AdapterException {
+    void testGetCapabilitiesWithExcludedCapabilitiesList() throws AdapterException {
+        setGenericSqlDialectProperty();
+        setDerbyConnectionProperties();
+        this.rawProperties.put(SCHEMA_NAME_PROPERTY, "SYSIBM");
+        this.rawProperties.put(EXCLUDED_CAPABILITIES_PROPERTY,
+              "ORDER_BY_EXPRESSION, LITERAL_NULL, FN_AGG_COUNT_STAR, FN_PRED_AND, FN_ADD");
+        final GetCapabilitiesRequest request =
+              new GetCapabilitiesRequest(TEST_DIALECT_NAME, createSchemaMetadataInfo());
+        final ExaMetadata exaMetadataMock = mock(ExaMetadata.class);
+        final GetCapabilitiesResponse response = this.adapter.getCapabilities(exaMetadataMock, request);
+        assertThat(response.getCapabilities().getMainCapabilities(),
+              not(contains(MainCapability.ORDER_BY_EXPRESSION, LiteralCapability.NULL,
```

Comment:
Negative test here is pointless. Please add more capabilities and turn this into a positive test.